src/main/kotlin/uk/gov/dluhc/printapi/exception/CertificateNotFoundException.kt

```diff
@@ -2,10 +2,10 @@ package uk.gov.dluhc.printapi.exception
 
 import uk.gov.dluhc.printapi.database.entity.SourceType
 
-class CertificateNotFoundException(eroId: String, sourceType: SourceType, sourceReference: String) :
-    RuntimeException(
-        if (sourceType == SourceType.VOTER_CARD)
-            "Certificate for eroId = $eroId and application id = $sourceReference not found"
-        else
-            "Certificate for eroId = $eroId, sourceType = $sourceType and $sourceReference not found"
-    )
+class CertificateNotFoundException : RuntimeException {
+    constructor(eroId: String, sourceType: SourceType, sourceReference: String) :
```

Comment:
Overloaded constructor - nice 👍 
I'd forgotten the syntax for that because we don't use it often ; so good to see a nice example of it 👍 
Thanks